values.yaml

```diff
@@ -267,6 +267,7 @@ scheduler:
 
 # Airflow webserver settings
 webserver:
+  allowPodLogReading: true
```

Comment:
Do we want this false by default in our chart maybe?

Reply:
@ashb it shouldn't matter since we have external log storage, but glad to do so if you feel it's more secure.